lib/opentracing/span.rb

```diff
@@ -40,11 +40,20 @@ def get_baggage_item(key)
       nil
     end
 
+    # ** Deprecated **
```

Comment:
should be `@deprecated` with a reason. See http://www.rubydoc.info/gems/yard/file/docs/Tags.md#deprecated